options/base_options.py

```diff
@@ -31,6 +31,9 @@ def initialize(self, parser):
         parser.add_argument('--ngf', type=int, default=64, help='# of gen filters in the last conv layer')
         parser.add_argument('--ndf', type=int, default=64, help='# of discrim filters in the first conv layer')
         parser.add_argument('--netD', type=str, default='basic', help='specify discriminator architecture [basic | n_layers | pixel]. The basic model is a 70x70 PatchGAN. n_layers allows you to specify the layers in the discriminator')
+        parser.add_argument('--netD2', type=str, default='none', help='specify discriminator architecture [basic | n_layers | pixel]. The basic model is a 70x70 PatchGAN. n_layers allows you to specify the layers in the discriminator')
+        parser.add_argument('--netD_pretrained', action='store_true', help='whether to use a pretrained model, available for non "basic" model only')
```

Comment:
Do we really have a use case for a pretrained D ?

Reply:
No we don't really use pretrained D.

The choice of architecture is done by `--netD` and `--netD2`.

Reply:
I don't see when we'd need a D to be pretrained. Because in general, we don't want it to fit quickly, nor overfit. What do you think of removing this option ?